### PR TITLE
Booking list layout on show

### PR DESCRIPTION
**Pull request by AJCJ1**
changes the layout on show

### CODE REVIEW

File: app/assets/stylesheets/components/_booking_card.scss [hunk 1]
```diff
@@ -1,4 +1,9 @@
 .booking-card-cust {
   width:auto;
+  height:fit-content;
+  padding:30px;
   box-shadow: 8px 6px 2px $black;
+  border-radius: 15px;
+  position:sticky;
+  top: 50px;
```
Comment (AJCJ1) [inline app/assets/stylesheets/components/_booking_card.scss:8]: styles booking card to display on right side of description, and sticks on scroll.

File: app/assets/stylesheets/components/_booking_item.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+}
+
+.button-confirm {
+  background-color: $green;
+}
+
+.button-edit {
+  background-color: $yellow;
+}
+
```
Comment (AJCJ1) [inline app/assets/stylesheets/components/_booking_item.scss:12]: gives each button correct colouring

File: app/assets/stylesheets/components/_index.scss [hunk 3]
```diff
@@ -9,3 +9,4 @@
 @import "carousel_arrows";
 @import "form";
 @import "booking_card";
+@import "booking_item";
```
Comment (AJCJ1) [inline app/assets/stylesheets/components/_index.scss:12]: custom styling for each booking item in component

File: app/assets/stylesheets/config/_colors.scss [hunk 4]
```diff
@@ -1,7 +1,7 @@
 // Define variables for your color scheme
 
 // For example:
-$red: #FD1015;
+$red: #ff6062;
```
Comment (AJCJ1) [inline app/assets/stylesheets/config/_colors.scss:4]: changed red to be less harsh, more modern.

File: app/assets/stylesheets/pages/_index.scss [hunk 5]
```diff
@@ -8,19 +8,21 @@
   margin: 0 auto;
   width:100;
   display: grid;
-  grid-template-columns: 1fr 1fr;
+  grid-template-columns: 1fr 1fr 1fr;
```
Comment (AJCJ1) [inline app/assets/stylesheets/pages/_index.scss]: 3 cards per row - i will be changing to 4 in next push

File: app/assets/stylesheets/pages/_index.scss [hunk 6]
```diff
@@ -8,19 +8,21 @@
...
   row-gap: 32px;
   align-items: center;
   justify-items: center;
 }
 
 
 .header-custom {
-  height:200px;
+  height:400px;
```
Comment (AJCJ1) [inline app/assets/stylesheets/pages/_index.scss:19]: changed banner to be larger

File: app/assets/stylesheets/pages/_index.scss [hunk 7]
```diff
@@ -29,17 +31,34 @@
...
 }
 
+.logo-brand-banner {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 75%;
+  width: 50%;
+}
+
```
Comment (AJCJ1) [inline app/assets/stylesheets/pages/_index.scss:42]: needs work, but will be changing SVG to be white, and remove duplicate button.

File: app/assets/stylesheets/pages/_show.scss [hunk 8]
```diff
@@ -1,8 +1,12 @@
...
-.description {
+.description-card {
   color: white;
-  border-radius: 10px;
+  border-radius: 15px;
   box-shadow: 8px 6px 2px $black;
+  margin: 15px;
+  flex-grow: 1;
+  gap:20px;
+  flex-wrap: nowrap;
```
Comment (AJCJ1) [inline app/assets/stylesheets/pages/_show.scss:9]: styling for description in show page.

File: app/assets/stylesheets/pages/_show.scss [hunk 9]
```diff
@@ -23,8 +27,18 @@
...
-  // h2 {
-  //   padding-top:5%;
-  // }
+}
+
+.dab-container {
+  width:75%;
+  height:25%;
+}
+
```
Comment (AJCJ1) [inline app/assets/stylesheets/pages/_show.scss:36]: organising with flex show page description

File: app/controllers/bookings_controller.rb [hunk 10]
```diff
@@ -49,7 +49,7 @@ def create
       @booking.user = current_user
       if @booking.save
         redirect_to car_path(@car.id)
-        flash.notice = "You have booked this car from the #{@booking.start_date} to the #{@booking.end_date}"
+        flash.notice = "You have booked this car from the #{@booking.start_date.strftime("%a %d %B %Y")} to the #{@booking.end_date.strftime("%a %d %B %Y")}"
```
Comment (AJCJ1) [inline app/controllers/bookings_controller.rb:52]: changed the alert to display readable dates just as the booking items do.

File: app/views/cars/index.html.erb [hunk 11]
```diff
@@ -1,4 +1,9 @@
 <div class="header-custom">
+  <div class="logo-brand-banner">
+    <%= image_tag "car-logo.svg", class:"svg-cust" %>
+    <h1> DriveMe </h1>
+  </div>
```
Comment (AJCJ1) [inline app/views/cars/index.html.erb:5]: adds logo, but not yet styled! will be doing this in next push.

File: app/views/cars/index.html.erb [hunk 12]
```diff
@@ -38,9 +43,9 @@
 
       <%# Text below image %>
       <div class="card-body">
-        <h5 class="card-title"><%= car.brand %> - <%= car.model %></h5>
-        <p class="card-text">A Lovely <%= car.brand %> in <%= car.color %></p>
-        <p><%= "Price per day £#{car.price}" %></p>
+        <h3 class="card-title"><%= car.brand.capitalize %> <%= car.model.capitalize %></h3>
+        <p class="card-text">A Lovely <%= car.brand.capitalize %> in <%= car.color %>.</p>
```
Comment (AJCJ1) [inline app/views/cars/index.html.erb:45]: capitalises each brand incase user adds uncapitalised version.

File: app/views/cars/index.html.erb [hunk 13]
```diff
@@ -38,9 +43,9 @@
...
+        <p><%= "£#{car.price}/Day" %></p>
```
Comment (AJCJ1) [inline app/views/cars/index.html.erb:46]: short and sweetens description

File: app/views/cars/show.html.erb [hunk 14]
```diff
@@ -32,53 +32,54 @@
       <% else %>
         <%= nil %>
       <% end %>
-      <%= link_to "Back to car list", :root, class:"btn btn-dark desc-buttons p-4"%>
+      <%= link_to "Back to Search", :root, class:"btn btn-dark desc-buttons p-4"%>
```
Comment (AJCJ1) [inline app/views/cars/show.html.erb:35]: as we will be implementing a search feature in the index, this page should generally be called 'search' to the user?

File: app/views/shared/_flashes.html.erb [hunk 15]
```diff
@@ -1,11 +1,11 @@
 <% if notice %>
-  <div class="alert alert-info alert-dismissible fade show m-1" role="alert">
+  <div class="alert border rounded-pill border-dark alert-dismissible fade show m-1 bg-white text-dark" role="alert">
     <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
     <%= notice %>
   </div>
 <% end %>
 <% if alert %>
-  <div class="alert alert-warning alert-dismissible fade show m-1" role="alert">
+  <div class="alert alert-warning rounded-pill alert-dismissible fade show m-1" role="alert">
```
Comment (AJCJ1) [inline app/views/shared/_flashes.html.erb:8]: makes alerts prettier

**Review by vdequercize — APPROVED** [review]

**Pull request merged**